libs/utils/target_script.py

```diff
@@ -89,7 +89,11 @@ def push(self):
         :type actions: list(str)
         """
 
-        actions = ['set -e'] + self.commands + ['set +e']
+        self._pid_file = self._target.path.join(self._target.working_directory, 
```

Comment:
That's actually what I do (line 116), but `_bg_shell.kill()` (line 133) does not stop kill the script on the target, just the shell on the host :confused: